plantain/jvm/src/main/scala/org/w3/banana/plantain/PlantainURIOps.scala

```diff
@@ -17,7 +17,7 @@ trait PlantainURIOps extends URIOps[Plantain] {
     uri.fragment
 
   def isPureFragment(u: Plantain#URI): Boolean =
-    u.scheme.isEmpty && u.authority.isEmpty && u.path.isEmpty && u.query.isEmpty && u.fragment.isDefined
+    u.scheme.isEmpty && u.authority.isEmpty && u.path.isEmpty && u.query().isEmpty && u.fragment.isDefined
```

Comment:
akka-http API changed, so isEmpty now is a method with two default parameters than I cannot call without brackets